README.rst

```diff
@@ -44,16 +44,33 @@ and a ``--replay=<file>`` can be used to re-run the tests from a previous run. F
     $ pytest -n auto --replay-record-dir=build/tests/replay
 
 This will generate files with the node id of each test executed by each worker, for example worker ``gw1`` will generate
-a file ``.pytest-replay-gw1.txt`` with contents like this::
+a file ``.pytest-replay-gw1.txt`` with contents like this (for pytest-replay versions <=1.1.0)::
 
     test_foo.py::test[1]
     test_foo.py::test[3]
     test_foo.py::test[5]
     test_foo.py::test[7]
     test_foo.py::test[8]
 
+Currently, ``pytest-replay`` is generating the output in a different manner as each
```

Comment:
I don't think we need to support the old style at all. 😁

Reply:
I am going to delete it then, thanks!